pkg/agent/conversation.go

```diff
@@ -785,14 +811,18 @@ func (c *Agent) handleMetaQuery(ctx context.Context, query string) (answer strin
 	case "tools":
 		return "Available tools:\n\n  - " + strings.Join(c.Tools.Names(), "\n  - ") + "\n\n", true, nil
 	case "session":
-		if s, ok := c.ChatMessageStore.(*sessions.Session); ok {
-			out, err := s.String()
-			if err != nil {
-				return "", false, fmt.Errorf("failed to get session string: %w", err)
-			}
-			return out, true, nil
+		if c.session == nil {
```

Comment:
we should never be in this state given that we will always have a session (defaulted to memory).